### ISSUE TITLE
Requesting the `Void` type hangs the UI

### ISSUE THREAD

**Issue by runarorama** [unisonweb/codebase-ui#150]
Searching for the string `Void` in the Find feature seems to spin forever.

Scrolling down to the `Void` type on the left nav bar and clicking it results in an entry in the content pane that looks like it's loading. It takes a minute or so to fail with an error. UCM is totally unresponsive while this is happening.

A query to the API like `api/getDefinitions?names=Void` responds normally and quickly.

**Comment by hojberg** [comment]
Yeah `GET /api/find?query=void` instantly causes this - very strange.
